system76/galp5-1650/default.nix

```diff
@@ -0,0 +1,27 @@
+{ config, lib, pkgs, ... }:
+{
+  imports = [
+    ../.
+    ../../common/gpu/nvidia/prime.nix
+    ../../common/gpu/nvidia/ampere
+  ];
+
+  boot.initrd.kernelModules = [ "nvidia" ];
+
+  hardware.graphics = {
+    enable = lib.mkDefault true;
+    enable32Bit = lib.mkDefault true;
```

Comment:
I am bit unsure if this is actually hardware configuration and not just about playing old games, which not everyone does:

```suggestion
```

Reply:
I'm pretty sure Steam will need it in general as it needs 32bit even if you have modern games.

Reply:
Ok, but than if we enable the module everywhere, should we not just do it in one place? i.e. nvidia.nix?